packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -278,14 +278,16 @@ export default mixins(
 			},
 		},
 		mounted() {
-			this.fullyExpanded = !this.isCollapsed;
 			if (this.$refs.user) {
 				this.$externalHooks().run('mainSidebar.mounted', { userRef: this.$refs.user });
 			}
 			if (window.innerWidth > 900 && !this.isNodeView) {
 				this.$store.commit('ui/expandSidebarMenu');
 			}
 			this.checkWidthAndAdjustSidebar(window.innerWidth);
+			Vue.nextTick(() => {
```

Comment:
again, just a nitpick, I prefer `await this.$nextTick()` (I'm super lazy and most of the time I chose what's less character or lines to write down)

so you can save a separate import for `Vue` but need to make `mounted` async

```suggestion
			await this.$nextTick();
			this.fullyExpanded = !this.isCollapsed;
```